doc/nrf/ug_nrf5340.rst

```diff
@@ -567,6 +567,27 @@ See the following instructions.
    :start-after: fota_upgrades_start
    :end-before: fota_upgrades_end
 
+Simultaneous multi-image DFU
+****************************
+
+The simultaneous update of multiple images is available for testing since |NCS| v1.7.0.
+It allows both the application core and the network core to be updated at the same run.
+
+To enable the simultaneous update of multiple images set the following options:
+
+* :kconfig:`CONFIG_BOOT_UPGRADE_ONLY` - The simultaneous update of multiple images does not support network core image reversion so it is recommended to disable application image reversion.
+* :kconfig:`CONFIG_PCD_APP` - Enable commands exchange with the network core.
+* :kconfig:`CONFIG_UPDATEABLE_IMAGE_NUMBER` - Enable support for multiple updatatable images by setting this option to ``2``.
```

Comment:
Maybe: `Enable simultaneous support...`
Update of both Application and Network images was supported in NCS already, just not simultaneously

Reply:
The current definition is correct if the same config is used for both simultaneous and separate updating of multiple images.

Though there is a typo: "updatatable" -> "updatable".

Reply:
I changed images to partitions. I am not sure we should say simultaneous here. This option allows multiple update slots to exist at the same time (if I understand it correctly) so it will allow simultaneous update in the end. But by itself it's not enough to make simultaneous update.

Reply:
typo is fixed